### PR TITLE
Upload-it: a demo Jupyter notebook extension

### PR DESCRIPTION
**Pull request by salikh**
Split off from the uploader pull request.

### CODE REVIEW

File: nbextensions/upload_it/main.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    const config = Jupyter.notebook.config;
+    for (let key in configuration) {
+      if (config.data.hasOwnProperty(key)) {
+        configuration[key] = config.data[key];
+      }
+    }
+  }
+
+  // Returns a JQuery object with the dialog.
+  function build_upload_dialog() {
```
Comment (yunabe) [inline nbextensions/upload_it/main.js]: ditto
https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers

File: nbextensions/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Jupyter notebook extensions
+
+This directory contains the Jupyter notebook extensions
+that are useful for the programming education assistant.
```
Comment (yunabe) [inline nbextensions/README.md]: Nice to have a link about Jupyter notebook extension development guide.
maybe this?
https://jupyter-notebook.readthedocs.io/en/stable/extending/frontend_extensions.html

Comment (salikh) [reply nbextensions/README.md]: Nice idea, thanks! Added the link.

File: nbextensions/upload_it/main.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (salikh) [reply nbextensions/upload_it/main.js]: Sorry for being sloppy. I think that sticking with Google's javascript style is a better choice going forward. Updated all identifiers.

**Comment by salikh** [comment]
Hi Yu, I have addressed the comments and now I am pushing this change as I see you "approved" stamp.

**Pull request merged**